requirements/install.pip

```diff
@@ -1 +1,2 @@
 udata>=2.1.4.dev
+requests
```

Comment:
Not sure if I should specify a version. Any version as a dependency of udata will do